content/millennium/r4/foundation/other/binary.md

```diff
@@ -9,11 +9,13 @@ title: Binary | R4 API
 
 ## Overview
 
-The Binary resource can contain any clinical content such as text, images, and PDFs. This resource is currently limited to Continuity of Care Documents (CCD), clinical documents, diagnostic reports, and communication message content. The supported diagnostic reports are Anatomic Pathology, Cardiology, Microbiology, and Radiology.
+The Binary resource can contain any clinical content such as text, images, and PDFs. This resource is currently limited to Continuity of Care Documents (CCD), clinical documents, diagnostic reports, and communication message content. 
+Example supported diagnostic report types include Anatomic Pathology, Cardiology, Microbiology, and Radiology.  
+_This list is not exhaustive and subject to change, as the list of clinical events that may qualify can be customized by clients_
 
-It is recommended to request Binary resources only after obtaining links to the resources via references from DiagnosticReport or DocumentReference. It is not recommended to start a workflow with Binary resources.
+It is recommended to request Binary resources only after obtaining links to the resources via references from [DiagnosticReport](http://fhir.cerner.com/millennium/r4/clinical/diagnostics/diagnostic-report) or [DocumentReference](http://fhir.cerner.com/millennium/r4/foundation/documents/document-reference). It is not recommended to start a workflow with Binary resources.
```

Comment:
is it possible to build internal reference links dynamically? If the R4 DocumentReference documentation moves from that URL, when this will break

Reply:
Spoke with George over IM about this.
We have identified links like these as something that will need to be updated in tandem with our hosting on the new site.

We'll be looking to complete that work over the coming sprints, and be tracking that work under separate stories.

For now, the links will work as intended to redirect users over to the correct endpoint.
We'd like to proceed with merging, understanding that these will be adjusted at a later date.